Android/Shared/ITMSamples.properties

```diff
@@ -0,0 +1,37 @@
+# suppress inspection "UnusedProperty" for whole file
```

Comment:
This file should not be checked in. (It should be in the .gitignore.)

Reply:
Fixed